syntax.css

```
@@ -0,0 +1,218 @@
+body {
+    font-family:arial;
+    margin=0px;
+    background-color: #eee
+}
+
+
+#Titre {
+    background-color:black;
+    color:white;
+    width: 100%;
+    height:auto;
+    text-align: center;
+}
+
+#Titre ul {
+    text-align: right;
+    margin-right: 10px;
+    margin-bottom: 0px;
+}
+
+#Titre ul li{
+    display:inline-block;
+    margin-left: 25px;
+}
+#Titre ul li a {
+    text-decoration: none;
+    color:white;
+    
+}
+
+#Titre ul li a:hover{
+    background-color: black;
+    color:#B59100;
+}
+
+#Intro img {
+    width:100%;
+    margin-top: 0px;
+}
+
+#Intro p{
+    text-align: center;
+    border-width: 2px;
+    border-style:groove;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    border-color:#B59100;
+    
+}
+
+
+
+#Histoire h2{
+    background-color: black;
+    height:auto;
+    width=100%;
+    margin-top: 0px;
+    margin-bottom: 0px;
+    color:white;
+    text-align: center;
+}
+#Histoire p {
+    margin-top: 10px;
+    margin-bottom: 10px;
+    text-align: justify;
+}
+
+#Histoire p a {
+    color:black;
+}
+
+#Histoire figure {
+    float: left;
+    margin-left: 5px;
+    margin-bottom: 5px;
+    margin-top: 5px;
+    margin-right: 10px;
+    font-style: italic;
+    font-size:12px;
+    text-align: center;  
+}
+
+#Histoire figure figcaption a {
+    text-decoration: none;
+    color: black;
+}
+
+#Histoire figure figcaption a:hover {
+    background-color: black;
+    color:white;
+}
+
+#Techniques h2 {
+    background-color:black;
+    color:white;
+    text-align:center;
+    margin-top: 5px;

```

Le changement ci-dessous fera la même chose, mais en un ligne au lieu de 4 ;) Le premier nombre c'est pour top et bottom et le deuxième pour left et right.
```suggestion
    margin: 5px 0px;
```